rsync.go

```diff
@@ -43,8 +43,8 @@ func Sync(via string, port uint, src, dst string, verbose bool) {
 
 	command := "rsync " + strings.Join(args, " ")
 
-	// fmt.Println("/bin/sh", "-c", command)
-	cmd := exec.Command("/bin/sh", "-c", command)
+	// fmt.Println("sh", "-c", command)
```

Comment:
Can we remove this dependency on sh? Or replace it by a corresponding multiplattfrom go library?